src/app.module.ts

```diff
@@ -2,6 +2,7 @@ import { Module } from "@nestjs/common";
 import { SequelizeModule } from "@nestjs/sequelize";
 import { TodosModule } from "./todos/todos.module";
 import { ConfigModule } from '@nestjs/config';
+import { Todo } from "./todos/todos.model";
 
```

Comment:
Differend code style. Somewhere " ", somewhere ' '.

Reply:
@skipGit please describe

Reply:
some of the imports i wrote by myself, but other were generated with autocomplete tools of my IDE. In JSlike languages there is no difference between "packagename" and 'packagename' in import section.